### PR TITLE
Creation of Exact Quantile Check

### PR DESCRIPTION
**Pull request by jmilis2000**
*Issue #, if available:*

*Description of changes:*
Added hasExactQuantile Check. Similar to hasApproxQuantile but finds more precise quantile value.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rdsharma26** [comment]
@jmilis2000 thanks for the PR! We will be having a look shortly. Can you please check in the meantime why the build is failing?

**Comment by jmilis2000** [comment]
> @jmilis2000 thanks for the PR! We will be having a look shortly. Can you please check in the meantime why the build is failing?

@rdsharma26 I fixed the issue. The build should work now. It failed because there was no amazon copyright at the top of the new analyzer i created.

**Review by rdsharma26 — APPROVED** [review]
LGTM. To confirm, you are using Spark's `percentile` to calculate the exact percentile rather than using the existing `hasApproxQuantile` check with error = `0.0`. I assume this is because the latter uses Spark's approx quantile method.

File: src/main/scala/com/amazon/deequ/analyzers/ExactQuantile.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.analyzers
+
+import com.amazon.deequ.analyzers.Preconditions.{hasColumn, isNumeric}
+import com.amazon.deequ.analyzers.Analyzers.{conditionalSelection, ifNoNullsIn}
+import com.amazon.deequ.metrics.FullColumn
+import org.apache.curator.shaded.com.google.common.annotations.VisibleForTesting
```
Comment (rdsharma26) [inline src/main/scala/com/amazon/deequ/analyzers/ExactQuantile.scala:22]: We use `import com.google.common.annotations.VisibleForTesting` in other places.

**Pull request merged**